### PR TITLE
feature/SM-1 Added the login screen

### PR DESCRIPTION
**Pull request by gabrieltunaru**

### CODE REVIEW

**Comment by alexandraveres7** [comment]
Please resolve conflicts!

File: src/loose.oose.fis.lab.student.manager/fxml/login.fxml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (alexandraveres7) [inline src/loose.oose.fis.lab.student.manager/fxml/login.fxml]: Remove unused file!

File: src/loose.oose.fis.lab.student.manager/controllers/LoginController.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package controllers;
```
Comment (alexandraveres7) [inline src/loose.oose.fis.lab.student.manager/controllers/LoginController.java]: Remove unused file!

File: src/loose.oose.fis.lab.student.manager/controllers/LoginController.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+            if (password == null || password.isEmpty()) {
+                loginMessage.setText("Password cannot be empty");
+                return;
+            }
+
+            if (username.equals("student") && password.equals("student")) {
+                loginMessage.setText("Logged in as student!");
+                return;
+            }
```
Comment (alexandraveres7) [inline src/loose.oose.fis.lab.student.manager/controllers/LoginController.java]: If login as a teacher succeeds, the user should be taken to the view-students scene”.

Comment (gabrieltunaru) [reply src/loose.oose.fis.lab.student.manager/controllers/LoginController.java]: trebuia la teacher nu la student :)

**Pull request merged**